Lib/asyncio/unix_events.py

```diff
@@ -1276,7 +1282,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     def __del__(self, _warn=warnings.warn):
         threads = [thread for thread in list(self._threads.values())
-                   if thread.is_alive()]
+                            if thread.is_alive()]
```

Comment:
This wouldn't be worth fixing it's own PR by any means and I can remove it if needed. It's just a formatting adjustment to align the contents of the list comprehension.

Reply:
I think the old formatting makes more sense. At least that's how I format my code. :)

Reply:
Ah okay, I can revert it back to the old formatting then.